pkg/scheduler/backend/queue/scheduling_queue.go

```diff
@@ -771,6 +774,16 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(logger klog.Logger, pInfo *
 		return fmt.Errorf("Pod %v is already present in the backoff queue", klog.KObj(pod))
 	}
 
+	if pInfo.UnschedulablePlugins.Union(pInfo.PendingPlugins).Len() == 0 {
+		// This Pod came back because of some unexpected errors (e.g., a network issue).
+		pInfo.ErrorCount++
+	} else {
+		// This Pod is rejected by some plugins, not coming back due to unexpected errors (e.g., a network issue)
+		pInfo.UnschedulableCount++
+		// We should reset the error count because the error is not gone.
```

Comment:
This comment is confusing to me. Because if "the error is not gone" mean we should keep `pInfo.ErrorCount`?
But according to the following changes, should we say some plugins in this scheduling cycle have rejected this pod, we ignore previous unexpected error counts because we prioritize the current unscheduable error.

Reply:
Sorry, it's just a mistake; I should've say "the error is gone"

Reply:
Got it, thanks